src/main/java/it/unisa/Amigo/gruppo/services/GruppoService.java

```diff
@@ -1,29 +1,31 @@
 package it.unisa.Amigo.gruppo.services;
 
-import it.unisa.Amigo.gruppo.domain.ConsiglioDidattico;
-import it.unisa.Amigo.gruppo.domain.Dipartimento;
-import it.unisa.Amigo.gruppo.domain.Persona;
-import it.unisa.Amigo.gruppo.domain.Supergruppo;
+import it.unisa.Amigo.gruppo.domain.*;
 
 import java.util.List;
 
 /**
  * Questa interfaccia definisce i metodi  per la logica di Business del sottositema "Gruppo"
  */
 public interface GruppoService {
```

Comment:
Stavo pensando di dividere in due il service, quello che ha a che fare strettamente con le persone, e quello che ha a che fare solamente con creazione di gruppi e commissioni. Il primo prenderebbe tutti i metodi che restituiscono persone e il secondo la creazione, l'aggiunta, la creazione e la nomina di responsabili.
**DA VALUTARE, NON SI DEVE FARE PER FORZA ORA**.